lighthouse-core/report/styles/report.css

```diff
@@ -50,6 +50,8 @@ span, div, p, section, header, h1, h2, li, ul {
   --report-width: 1280px;
   --report-menu-width: 280px;
   --report-header-height: 58px;
+
+  --config-panel-header-height: 50px;
```

Comment:
css once added is difficult to unwind after the fact, so re: above comment, I think a lot of this would be better living in a separate file explicitly for perfX UI, even if it introduces some redundancies.

We're also going to have to think about maintainability, as there are a few proposals about overhauling lighthouse scoring this quarter which will necessitate some level of report structure changes. No one else is really running the performance experiment yet, so we'll need to come up with a to make sure we don't break it.

(this and viewer make me increasingly want to set up a selenium check of the report in travis)